download.go

```diff
@@ -29,6 +48,32 @@ func downloadFile(path string, url string) (err error) {
 	return err
 }
 
+func gitDownload(url string, path string, name string) error {
+	_, err := os.Stat(filepath.Join(path, name, ".git"))
+	if os.IsNotExist(err) {
+		err = passToGit(path, "clone", url, name)
+		if err != nil {
+			return fmt.Errorf("error cloning %s", name)
+		}
+
+		return nil
+	} else if err != nil {
```

Comment:
No need for this `else` (and the next one), since the blocks end with a `return`.